src/System.Net.Sockets.Legacy/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -1443,12 +1445,14 @@ public void Socket_ReceiveFromDnsEndPoint_Throws()
         }
 
         [Fact]
+        [ActiveIssue(DummyOSXPacketInfoIssue, PlatformID.OSX)]
```

Comment:
Some will remain unsupported, but some may be fixable. The silver lining w.r.t. the tests that are not supportable is that existing code should be handling failure cases already, as the API does not strictly guarantee the delivery of the missing data.